docset/winserver2019-ps/tls/Get-TlsCipherSuite.md

```diff
@@ -11,7 +11,7 @@ title: Get-TlsCipherSuite
 # Get-TlsCipherSuite
 
 ## SYNOPSIS
-Gets the list of cipher suites for TLS for a computer.
+Gets the cipher suites for TLS for a computer.
```

Comment:
```suggestion
Gets the list of TLS cipher suites available on the computer.
```

Reply:
@doctordns - please commit the suggested change as per @tseward's request. Many thanks.